source/linux/duplicates/Registry.cpp

```diff
@@ -1,5 +1,8 @@
 #include "StdAfx.h"
 #include "Registry.h"
+#include "Common.h"
+#include "Card.h"
```

Comment:
I am not sure why you need these.
They are already included by Registry.h

Reply:
Unless you want them explicit.

Fine.

Reply:
Ahh, I think I see what's going on. It's picking source/linux/registry.h instead of source/Registry.h. Similar problem to:

```
/Users/sh95014/Developer/Mariani/source/linux/tape.cpp:6:10: warning: non-portable path to file '"tape.h"'; specified path differs in case from file name on disk [-Wnonportable-include-path]
#include "Tape.h"
         ^~~~~~~~
         "tape.h"
1 warning generated.
```

Reply:
is XCode case insensitive?

I should have chosen better names... I dont mind removing the ambiguity anyway.

Reply:
Not Xcode specifically, but macOS in general. I'll make another PR then. Let me know if you have suggestions for names. I think `linux/tape.h` should become `linux/CassetteTape.h` to match the class name, but I don't have a good idea for `linux/registry.h`.

Reply:
https://github.com/audetto/AppleWin/pull/90